bank/bank-backend/package.json

```diff
@@ -32,15 +32,14 @@
   },
   "devDependencies": {
     "@dbos-inc/dbos-cloud": "^1.15.9",
-    "@dbos-inc/eslint-plugin": "^1.1.5",
+    "@dbos-inc/eslint-plugin": "^3.0.0",
     "@types/jest": "^29.5.3",
     "@types/koa__cors": "^4.0.1",
     "@types/koa__router": "^12.0.1",
     "@types/koa-bodyparser": "4.3.10",
     "@types/koa-logger": "^3.1.2",
     "@types/lodash": "^4.14.197",
     "@types/supertest": "^2.0.14",
-    "eslint": "^8.56.0",
```

Comment:
I think it's better to explicitly specify an eslint version here, because for some npm versions peerDependencies are not installed automatically.

https://nodejs.org/en/blog/npm/peer-dependencies
> UPDATE: npm versions 1, 2, and 7 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For npm versions 3 through 6, you will receive a warning that the peerDependency is not installed instead.